client/src/components/pages/TeacherProfile/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import './index.css';
+import TeacherImage from '../../../assets/teacher.jpg';
+import fakeData from './fakeData';
+
+class TeacherProfile extends Component {
+  state = {
+    data: {},
+    active: 1,
+  };
+
+  componentDidMount() {
+    this.setState({ data: fakeData.teacher1 });
+    // we will make a request to get data by using axios, above line is a fake data
+  }
+
+  handelerActive = num => {
+    this.setState({ active: num });
+  };
+
+  render() {
+    const { active } = this.state;
+    const { data } = this.state;
+
+    return (
+      <div className="teach-profile">
+        <div className="teach-profile__top">
+          <div className="box-img">
+            <img src={TeacherImage} alt="teacher img" />
+          </div>
+          <h3 className="teacher-name">{data.name}</h3>
+          <p className="teacher-subject">{data.subject} Teacher</p>
+        </div>
+        <div className="teach-profile__list">
+          <button
+            type="button"
+            onClick={() => this.handelerActive(1)}
```

Comment:
Instead of magic values that don't say anything use either:
- constants: `const var CONTENT_ABOUT = 1;`
- or enums:
```
var ActiveContentEnum = Object.freeze({
  ABOUT: 1,
  CONTACT: 2,
  INTERESTS: 3,
});
```
I would also rename the `handlerActive` function to for example `showContent`, so in the end it would look like: `onClick={() => this.showContent(ActiveContentEnum.ABOUT)}` which is pretty self-explanatory (maybe even better names would be activeSection, ActiveSectionEnum and showSection())

Reply:
After discussion we suggest other solution with the point, it similar this
It's very good if you give your feedback about it, and what's the best, I'm looking forward to your opinion

thank you so much